src/additional/maxfiles.js

```diff
@@ -0,0 +1,16 @@
+//Limit the number of files in a FileList.
```

Comment:
Just for consistency with the other comments in this project, can you add a space before the comment?
The same goes for other comments bellow.